src/main/kotlin/app/revanced/cli/command/utility/InstallCommand.kt

```diff
@@ -30,7 +31,7 @@ internal object InstallCommand : Runnable {
 
     @Option(
         names = ["-m", "--mount"],
-        description = ["Mount the supplied APK file over the app with the supplied package name."],
+        description = ["Mount the supplied APK file over the app with the given package name."],
```

Comment:
I think we use "supply" in multiple places in this repo for consistency reasons. Using "given" expands the terminology and introduces inconsistency. Either apply this change everywhere where the terminology "supply" is used, or keep it "supply"

Reply:
I have no issues with the original, just personally dislike repeating same word is all.